readthedocs/urls.py

```diff
@@ -86,9 +86,13 @@
 
 if 'readthedocsext.donate' in settings.INSTALLED_APPS:
     # Include donation URL's
-    groups.append([
-        url(r'^sustainability/', include('readthedocsext.donate.urls')),
-    ])
+    groups.insert(0,
+                  [url(r'^sustainability/', include('readthedocsext.donate.urls'))]
+                  )
+    for num, _url in enumerate(rtd_urls):
```

Comment:
Same question again. This is only if the donate app is installed, why? I don't find the relationship between donate app and search app.